packages/backend/src/graphql/resolvers/liquidity.test.ts

```diff
@@ -1516,7 +1516,7 @@ describe('Liquidity Resolvers', (): void => {
       })
       payment = await createOutgoingPayment(deps, {
         accountId,
-        receiver: `${Config.publicHost}/${uuid()}/incoming-payments/${uuid()}`,
+        receiver: `${Config.adminUrl}/${uuid()}/incoming-payments/${uuid()}`,
```

Comment:
should this be the `open payments url`?

Reply:
it's an old commit, already changed to `openPaymentsHost` [here](https://github.com/interledger/rafiki/pull/497/files#diff-9054f64f0e8dcb9aef3f4619ab9f14fb3654eb010f002c6d934d5678a812119fR1518).